packages/e2e-tests/specs/adding-blocks.test.js

```diff
@@ -6,11 +6,13 @@ import {
 	insertBlock,
 	getEditedPostContent,
 	pressKeyTimes,
+	switchToEditMode,
 } from '@wordpress/e2e-test-utils';
 
 describe( 'adding blocks', () => {
 	beforeEach( async () => {
 		await createNewPost();
+		await switchToEditMode();
```

Comment:
Is this something temporary?

Reply:
No, it's not because by default we're now in navigation mode and most of our tests assume edit mode.